themes/godotengine/i18n/po/messages.po

```diff
@@ -257,3 +229,31 @@ msgstr ""
 #: godotengine/partials/footer.htm:9
 msgid "contributors"
 msgstr ""
+
+#: godotengine/layouts/home.htm:324
```

Comment:
That diff is interesting as it shows that the PO generation tool doesn't produce a deterministic sorted list of strings, it adds new ones at the end.

Reply:
Not sure how they are generated, I just press the buttons on the CMS 😅 but maybe generating them alphabetically might be a solution

Reply:
Will be fixed by https://github.com/godotengine/godot-website/pull/494.